test/helpers/utils.js

```diff
@@ -0,0 +1,18 @@
+import Q from 'q';
+import _ from 'lodash';
+
+export const iframe = () => {
```

Comment:
Rename the file as `iframe` and change this to `export default function iframe() {`?